README.md

```diff
@@ -16,6 +16,16 @@
 
 A local testnet for Starknet... in Rust!
 
+### Features
+
+- Forking - interact with contracts deployed on mainnet or testnet
```

Comment:
Should we add `[links](...)` to documentation pages for each of these list items?